backend/src/main/java/ai/verta/modeldb/project/FutureProjectServiceImpl.java

```diff
@@ -63,27 +66,80 @@ public void updateProjectDescription(
   public void addProjectAttributes(
       AddProjectAttributes request,
       StreamObserver<AddProjectAttributes.Response> responseObserver) {
-    super.addProjectAttributes(request, responseObserver);
+    try {
+      final var response =
```

Comment:
I like to name future variables `futureResponse` for my own sanity; I think it helps.

Reply:
Fixed